packages/react-ui-ag/src/Listings/DesktopListing.js

```diff
@@ -30,7 +30,7 @@ export default class DesktopListing extends PureComponent {
 
   get renderInfo() {
     const { theme, listing, ratings, propertyName } = this.props
-    const { singleFamily, rating, lastUpdated, phone } = listing
+    const { singleFamily, rating, ulaText, phone } = listing
```

Comment:
new consolidated prop which will be comprised of a selector that returns `ula ? displayUla : DisplayLastUpdated`